website/docs/Best_practices/reusing_code.md

```diff
@@ -0,0 +1,23 @@
+
+### Code WDL tasks in external WDL scripts, not in workflow definitions
+When coding extensive WDL workflows that reuse multiple tasks, it makes sense to place those tasks in separate WDL scripts that your workflow(s) can import. This keeps your workflows clean and easier to read. It also allows you to choose how you group tasks together for reusability and maintenance. For example, you might choose to group together tasks that are for a specific workflow or group tasks by function. In either case, you can put tasks into a separate WDL script dedicated to the tasks alone. WARP has several examples of these task WDLs that you can use as a template, such as the [Imputation task WDL](https://github.com/broadinstitute/warp/blob/master/tasks/broad/ImputationTasks.wdl), which groups tasks that are for the Imputation pipeline, or the [Utilities task WDL](https://github.com/broadinstitute/warp/blob/master/tasks/broad/Utilities.wdl), which contains common genomic sequencing file conversion and validation tasks that are applied across multiple workflows. 
+
+## Reusing someone else’s WDL code
+The best way to reuse WDL code written by someone else depends on how you want to implement and update it for your workflows. You can reuse code exactly as written by the contributor or you can modify it for your own purposes. If you plan to use code as-is, it can be helpful to pull that code directly from whatever repository stores it. For example, if it’s in GitHub, you can either fork the repository or point directly to code using raw, full-length `http` paths (see Cromwell’s documentation on imports). You can also use forking if you’re planning to modify code, but in that case, sometimes it might even be easier to copy and paste the code. No matter your intentions, keep the following tips in mind when using external code.
+
+### Decide if you want to keep the code up-to-date with the source
+Sometimes you might want to reuse code that is regularly updated by external contributors. If you want to keep up with these updates, you don’t want to deal with the hassle of manually making modifications. This is why the Pipeline Development team recommends importing the WDL code directly from wherever it lives in the cloud. If you’re only trying to reuse a specific version of WDL code, you can still pull it directly, but you’ll want to pin to the specific code version.
```

Comment:
Might change "wherever it lives in the cloud" to "wherever it lives" or "wherever it is maintained". I think mostly this will be github or some other version control platform. With cloud I usually think Google cloud, Azure, or AWS which is not really where WDL is typically maintained

Reply:
Good point! I updated this sentence. Thank you!